.github/workflows/optimize-deploy-artifacts.yml

```diff
@@ -76,15 +76,15 @@ jobs:
       with:
         compose_file: .github/actions/compose/docker-compose.smoketest.yml
         project_name: smoketest
+        timeout: 600
```

Comment:
we need to reduce this timeout to something reasonable

Reply:
actually we can remove this param here completely, the default value of 300s works fine